src/script/lua_api/l_env.h

```diff
@@ -116,7 +116,7 @@ class ModApiEnvMod : public ModApiBase {
 	// get_day_count() -> int
 	static int l_get_day_count(lua_State *L);
 
-	// find_node_near(pos, radius, nodenames) -> pos or nil
+	// find_node_near(pos, radius, nodenames, search_centre) -> pos or nil
```

Comment:
Yes on search code uses 'center' throughout. 'centre' is only used occasionally in docs and comments (mostly by me).